concourse/pipelines/guest-package-build.yaml

```diff
@@ -206,7 +206,7 @@ jobs:
       - task: upload-guest-agent-deb11
         file: guest-test-infra/concourse/tasks/gcloud-package-operation.yaml
         vars:
-          package_path: guest-agent/google-guest-agent_((.:package-version))-g1_amd64.deb
+          package_path: guest-agent/google-guest-agent_((.:package-version))-g1_amd64.deb;guest-agent/google-guest-agent_((.:package-version))-g1_arm64.deb
```

Comment:
i think we decided to use a real list in the RPC rather than a delimiter

Reply:
yes. it's constructed in gcloud-package-operation.yaml now, since we don't want to put a complex json string here

Reply:
this still has semicolons, is that right?

Reply:
modified to directly passing in json string, no more